tests/pom.xml

```diff
@@ -64,6 +64,11 @@
             <artifactId>spark-rapids-jni</artifactId>
             <classifier>${jni.classifier}</classifier>
         </dependency>
+        <dependency>
+            <groupId>com.nvidia</groupId>
+            <artifactId>rapids-4-spark-hybrid_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
```

Comment:
Please use the version of hybrid itself like other places:
```xml
        <dependency>
            <groupId>com.nvidia</groupId>
            <artifactId>rapids-4-spark-hybrid_${scala.binary.version}</artifactId>
            <version>${spark-rapids-hybrid.version}</version>
            <scope>provided</scope>
        </dependency>
```
Like private jar, it has its own version.
During the release process, the `project.version` is not always equal to `spark-rapids-hybrid.version`.

And set scope as `<scope>test</scope>`?

Reply:
nice catch

Reply:
updated. I created an item for hyrid jar in the `dependencyManagement` section in the root pom to unify its version for sub modules.